.github/workflows/qml-label-check.yml

```diff
@@ -0,0 +1,24 @@
+name: QML Label component check
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run label component count
+        run: |
+          MATCHESRESULT=$(grep -r -e "^\s*Label\s*[{]" . | wc -l | sed 's/ //g')
```

Comment:
@claucambra Won't we have a false-positive in some other source files that are **non-.qml**, at some point, if the keyword **Label** is used elsewhere? Also, won't it be faster to run this step if we only run it for **.qml** files?

Reply:
Good points, changed so we only check QML files